server/src/main/java/org/elasticsearch/search/aggregations/PipelineAggregationBuilder.java

```diff
@@ -216,6 +209,11 @@ public void addBucketPathValidationError(String error) {
          */
         public abstract void validateParentAggSequentiallyOrdered(String type, String name);
 
+        /**
+         * Validates that the parent is sequentially ordered.
```

Comment:
There's a method right above that has the same javadoc.  I know these two methods don't do the same thing, so they should explain how they're different.

Reply:
:+1: I meant to do this but skipped it. thanks.